deps/build-deps-osx.sh

```diff
@@ -24,6 +39,10 @@ rm -rf ffmpeg
 
 if [ ! -d "$SRCDIR" ]; then
     git clone https://github.com/discord/lilliput-dep-source "$SRCDIR"
+
+    # TODO: Fix this in the upstream deps repo
+    cd "$SRCDIR"
+    wget -q -O giflib-5.2.2.tar.gz https://sourceforge.net/projects/giflib/files/giflib-5.2.2.tar.gz/download
```

Comment:
seems like 5.2.2 has been added in the upstream deps repo already and that discord has added a patch for giflib as well

Reply:
@remiolivier-discord I noticed you updated the deps repo, could you also update the build scripts in the main branch for lilliput? 🙏